src/rgw/driver/rados/rgw_gc.cc

```diff
@@ -51,7 +51,7 @@ void RGWGC::initialize(CephContext *_cct, RGWRados *_store) {
     op.create(false);
     const uint64_t queue_size = cct->_conf->rgw_gc_max_queue_size, num_deferred_entries = cct->_conf->rgw_gc_max_deferred;
     gc_log_init2(op, queue_size, num_deferred_entries);
-    store->gc_operate(this, obj_names[i], &op);
+    store->gc_operate(this, obj_names[i], &op, null_yield);
```

Comment:
you added an `optional_yield y` argument to `RGWGC::initialize()`, so you can pass that here. same with `RGWGC::send_split_chain()` and several other functions in rgw_gc.cc